formats/hocon/src/test/kotlin/kotlinx/serialization/hocon/HoconDurationDeserializerTest.kt

```diff
@@ -0,0 +1,71 @@
+package kotlinx.serialization.hocon
+
+import kotlin.time.*
+import kotlin.time.Duration.Companion.days
+import kotlin.time.Duration.Companion.hours
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.minutes
+import kotlin.time.Duration.Companion.seconds
+import kotlinx.serialization.*
+import org.junit.Assert.*
+import org.junit.Test
+
+class HoconDurationDeserializerTest {
+
+    @Serializable
+    data class Simple(val d: Duration)
+
+    @Serializable
+    data class Nullable(val d: Duration?)
+
+    @Serializable
+    data class Complex(val i: Int, val s: Simple, val n: Nullable, val l: List<Simple>, val ln: List<Nullable>, val f: Boolean)
```

Comment:
I'd also add `List<Duration>` and `Map<String, Duration>` to this test. AFAIR, `Map<Duration, Duration>` is prohibited?

Reply:
Made changes to support `List<Duration>` and `Map<String, Duration>`. `Map<Duration, Duration>` also works.